.circleci/config.yml

```diff
@@ -231,25 +231,25 @@ workflows:
           filters:
             branches:
               ignore: gh-pages
-      - test-docs:
-          filters:
-            branches:
-              ignore: gh-pages
-      - deploy-docs:
-          requires:
-            - test-docs
-          filters:
-            branches:
-              only: development
-      - build-tari-release:
-          filters:
-            branches:
-              ignore: gh-pages
-      - test-tari-release:
-          filters:
-            branches:
-              ignore: gh-pages
-      - clippy:
-          filters:
-            branches:
-              ignore: gh-pages
+      # - test-docs:
+      #     filters:
+      #       branches:
+      #         ignore: gh-pages
+      # - deploy-docs:
+      #     requires:
+      #       - test-docs
+      #     filters:
+      #       branches:
+      #         only: development
+      # - build-tari-release:
+      #     filters:
+      #       branches:
+      #         ignore: gh-pages
+      # - test-tari-release:
+      #     filters:
+      #       branches:
+      #         ignore: gh-pages
+      # - clippy:
+      #     filters:
+      #       branches:
+      #         ignore: gh-pages
```

Comment:
Why are these commented out?
Rather delete them or leave a comment as to why they are commented out.

Reply:
Yeah happy to delete them but @mikethetike mentioned just commenting them out.